src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.DesignTime.targets

```diff
@@ -47,20 +47,6 @@ Copyright (c) .NET Foundation. All rights reserved.
     <ProjectCapability Include="SupportsTypeScriptNuGet" />
   </ItemGroup>
 
-  <!--
-    For now we need to treat component files as if they have a single file generator. This will allow us
-    to trigger a workspace update for the declaration files when they change.
-  -->
-  <ItemGroup>
```

Comment:
Shoot we need to make this dependent on > 3.0 projects. Otherwise if a user downloads latest VS it will not properly work with older projects.